lua/obsidian/command.lua

```diff
@@ -70,12 +72,13 @@ end
 command.new = function(client, data)
   ---@type obsidian.Note
   local note
+  local open_in = util.get_open_strategy(client.opts.open_notes_in)
   if data.args:len() > 0 then
-    note = client:new_note(data.args)
```

Comment:
I suspect the change from `data.args` to `data.args[1]` has just broken the functionality of the `:ObsidianNew` command. If I call `:ObsidianNew Example`, then:
data.args = "Example"                                                                                                      
data.args[1] = nil
which breaks `note_id_func` since it never recieves the title.

Reply:
Ah that's a remnant of my first pass when I tried to do this via additional arguments, must have missed this change, will open a PR for a fix